benefits/oauth/views.py

```diff
@@ -24,7 +24,8 @@
 def login(request):
     """View implementing OIDC authorize_redirect."""
     verifier = session.verifier(request)
-    oauth_client = oauth.create_client(verifier.auth_provider.client_name)
+
+    oauth_client = register_provider(oauth, verifier.auth_provider)
```

Comment:
This looks good but I think we also need this change in the `authorize` function down below.

Reply:
I don't think it's needed since `authorize` always follows `login` currently, but it also doesn't hurt to have it in `authorize`. Made the change in [770f8e1](https://github.com/cal-itp/benefits/pull/2152/commits/770f8e185a192f802f1bec47b31c5b6e48a2b763)